src/renderer/components/caption-studio/TextEditor.vue

```diff
@@ -154,6 +154,7 @@ export default {
     EventBus.$on('caption_reset', this.reset);
     EventBus.$on('json_errors', this.onJsonErrors);
     this.$refs.Quill?.quill?.on('text-change', this.onEdit);
+    //this.$refs.Quill?.$el.addEventListener('paste', catchPasteEvent);
```

Comment:
Another thing that should be removed

Reply:
I think this line was something we wanted to come back to and evaluate later. It was causing errors, but we still want to find another way to add the event listener - there may already be a ticket about it. Does this ring a bell @902seanryan ? 

Either way I will remove the comment, just wanted to call that out!

Reply:
That sounds right